deepchem/pages/index.js

```diff
@@ -0,0 +1,264 @@
+import React, { useState, useEffect } from "react";
+import Link from "next/link"
+
+import Snackbar from "@mui/material/Snackbar";
+import MuiAlert from "@mui/material/Alert";
+import { TypeAnimation } from "react-type-animation";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { Tooltip as ReactTooltip } from "react-tooltip";
+import Bounce from 'react-reveal/Bounce';
+
+import BouncingAtoms from "../components/Home/BouncingAtoms/BouncingAtoms";
+import CustomCarousel from "../components/CustomCarousel/CustomCarousel";
+import CarouselItem from "../components/CustomCarousel/CarouselItem";
+import ExploreCardItem from "./../components/Home/ExploreCardItem";
+
+import atom from "../public/images/used-by/atom.png";
+
+import bist from "../public/images/used-by/bist.png";
+import csir from "../public/images/used-by/csir.png";
+import expedia from "../public/images/used-by/expedia.png";
+import firmenich from "../public/images/used-by/firmenich.png";
+import german from "../public/images/used-by/german-research-center-for-env-health.png";
+import ghddi from "../public/images/used-by/ghddi.png";
+import ibm from "../public/images/used-by/ibm-watson.png";
+import icms from "../public/images/used-by/icms.png";
+import jiangsu from "../public/images/used-by/jiangsu.png";
+import mit from "../public/images/used-by/mit.png";
+import northwestern from "../public/images/used-by/northwestern.png";
+import riken from "../public/images/used-by/riken.png";
+import skoltech from "../public/images/used-by/skoltech.png";
+import sri from "../public/images/used-by/sri-international.png";
+import strath from "../public/images/used-by/strath.png";
+import ucf from "../public/images/used-by/ucf.png";
+import uni_alberta from "../public/images/used-by/uni-alberta.png";
+import uni_bristol from "../public/images/used-by/uni-bristol.png";
+import uni_catalonia from "../public/images/used-by/uni-catalonia.png";
+import uni_central from "../public/images/used-by/uni-central-south.png";
+import uni_east from "../public/images/used-by/uni-east-china.png";
+import uni_geneva from "../public/images/used-by/uni-geneva.png";
+import uni_hunan from "../public/images/used-by/uni-hunan.png";
+import uni_macau from "../public/images/used-by/uni-macau.png";
+import uni_shanghai from "../public/images/used-by/uni-shanghai.png";
+import uni_taiwan from "../public/images/used-by/uni-taiwan.png";
+import uni_tartu from "../public/images/used-by/uni-tartu.png";
+import uni_tokyo from "../public/images/used-by/uni-tokyo.png";
+import uni_vienna from "../public/images/used-by/uni-vienna.png";
+import uni_zhejiang from "../public/images/used-by/uni-zhejiang.png";
```

Comment:
Doing explicit imports for each logo will make this hard to update. Can we instead have a way that allows us to add new logos without changing the js file?

Can we add documentation on how we would add a new logo?

Reply:
I have changed the way logos are imported now. To add new logos, now, they need to be simply added to the respective assets folder. The code does the rest and imports all the images from those folders. Also, I have added elaborate instructions to the website readme. Is that fine?